torat_server/client_ishell.go

```diff
@@ -257,10 +262,25 @@ func (client *activeClient) Shred(c *ishell.Context) {
 	}
 }
 
-// ping something?
-// TODO: I don't think this feature works
-func (client *activeClient) Ping(c *ishell.Context) error {
-	return client.RPC.Call("API.Ping", void, &void)
+// Speedtest
+func (client *activeClient) Speedtest(c *ishell.Context) {
+	c.ProgressBar().Indeterminate(true)
+	c.ProgressBar().Start()
+
+	r := shared.Speedtest{}
+	if err := client.RPC.Call("API.Speedtest", void, &r); err != nil {
+		c.ProgressBar().Final(yellow("["+client.Client.Name+"] ") + red("[!] Could not perform speedtest on client!"))
```

Comment:
Also print the actual err for added context.

Reply:
Okay